Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
I still think that cluttering the menu bar with a <kbd>↑</kbd>/<kbd>↓</kbd> alternate is overkill. I mean, at least on a discoverability standpoint, it has a better chance of being found than say, <kbd>⌥⌘A</kbd> for "Amend…" 😉.

As an (non-obvious) example, our beloved Finder uses <kbd>⌘↑/↓</kbd> to "open-parent/open-object", and adding ⌥ ~~makes it persist windows~~ is broken since Apple took away the ability to manage windows the way did. And it has no menu item anywhere.